app/services/users.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require('bcrypt');
+const errors = require('../errors');
+const logger = require('../logger');
+const { User } = require('../models');
+
+const createParams = params =>
```

Comment:
You're still mixing 3 things into one: `bycrypt` (could be a separated service of user because i can use that for other purposes not only password), `user service` (this is the actual file), `mapping` (this map the external api params and should be performed in the controller, all the params.blah should not be necessary here)